example_node:
  ros__parameters:
    example_parameter: world
    heartbeat_period: 200
example_node1:
  ros__parameters:
    example_parameter: world
    heartbeat_period: 200